tests/test_verilog/test_inline.py

```diff
@@ -0,0 +1,97 @@
+import magma as m
```

Comment:
Think these tests should just check json instead of verilog

Reply:
But checking the verilog verifies the translation through coreir (namely, the coreir backend reading the metadata correctly)

Reply:
I guess we could assume that the coreir test suite covers this